js/views/components/ModeratorsStatus.js

```diff
@@ -28,6 +29,19 @@ export default class extends BaseVw {
     };
   }
 
+  setState(state = {}, options = {}) {
+    const combinedState = { ...this.getState(), ...state };
+    // Any time the state is set to loading, set the spinner timer if needed.
+    if (state.loading && combinedState.showSpinner) {
```

Comment:
Can we please clear the timeout here before creating a new one to ensure multiple timeouts aren't queue up?